docs/changehistory/NextVersion.md

```diff
@@ -4,11 +4,13 @@ publish: false
 
 # NextVersion
 
+
 Table of contents:
 
 - [Selection set](#selection-set)
 - [Font APIs](#font-apis)
 - [API deprecations](#api-deprecations)
+  - [@itwin/ui-appui-abstract](#itwinui-appui-abstract)
```

Comment:
1. we should order these alphabetically
2. this is not an itwinui package, this is itwin/appui-abstract